precipitation/ngsild-payloads/Groundwater.json

```diff
@@ -0,0 +1,31 @@
+{
+   "id": "urn:ngsi-ld:Groundwater:Pegomas",
+   "type": "Groundwater",
+   "waterLevel": [
+      {
+         "type": "Property",
+         "observedBy": {
+            "type": "Relationship",
+            "object": "urn:ngsi-ld:Device:Station09995X0028/F"
+         },
+         "value": 0.0,
+         "unitCode": "MQS",
```

Comment:
just make sure about use of units: MQS: meter cubic per second. I'm not sure if it corresponds to WaterLevel.